docs-gen/content/rule_set/private_branch.md

```diff
@@ -1,54 +0,0 @@
----
-title: "Private Branch"
```

Comment:
What is our view on the Private branch. With the overlay functionality OEMs can easily add data to existing branches, but do we actually see the private branch as obsolete, or do we still see `Private` as a reasonable naming if you are to add signals that currently do not have any "logic" home in VSS. I can see two options and I have not yet made up my mind:

- Either we see "private" as  a good naming convention for extensions, and then we can briefly mention that in the "overlay.md" document, e.g. "If you are to add OEM-specific signals that does not fit into the VSS structure, consider adding them to Vehicle.Private.Something.
- Or we see "private" as obsolete, but then we should preferably remove "private" from the existing vspec files.

Reply:
I would see it as obsolete. I wanted to delete as well the folder in the spec file, but forgot. I described it as one scenario for overlay. But let's discuss it in the call later.

Reply:
General opinion leads towards removing private as obsolete:
- @danielwilms create issue for tools to remove old "private" code.
- @danielwilms remove empty private folder